courses/machine_learning/deepdive2/feature_engineering/solutions/Basic_Feature_Engineering_in_Keras.ipynb

```diff
@@ -0,0 +1,2721 @@
+{
```

Comment:
<p>Sorry, I should have warned you. We don't need the extra line at the end of the cell. I know the PEP8 checker flags it, but the checker assumes we're passing it a full .py file as opposed to a python notebook cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>No worries.  I'll take care of the other cells.  :)</p>

Reply:
I removed all the extra lines from all of the Notebooks.